src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -313,7 +316,10 @@ const ActionsPageEvent = ({
   const colonyNativeToken = tokens.find(
     ({ address }) => address === nativeTokenAddress,
   );
-
+  const decimalStakeAmount = getFormattedTokenValue(
+    values?.stakeAmount || 0,
+    getTokenDecimalsWithFallback(colonyNativeToken?.decimals),
```

Comment:
i think it would be better to pass just decimals in all of those places and use getTokenDecimalsWithFallback in getFormattedTokenValue util itself